spec/models/censor_rule_spec.rb

```diff
@@ -143,6 +143,40 @@
 
   end
 
+  describe '#expire_requests' do
+
+    it 'calls expire on the request if it is a request rule' do
```

Comment:
Super minor nitpick, betterspecs.org (which I think we've been following elsewhere) suggests to use contexts for this kind of thing: http://www.betterspecs.org/#contexts